_config.yml

```diff
@@ -79,9 +79,16 @@ index_old_doc_paths: false
 #
 distros:
   - 'bouncy'
+  - 'melodic'
+  - 'lunar'
+  - 'kinetic'
+  - 'indigo'
+
 old_distros:
   - 'ardent'
-
+  - 'jade'
```

Comment:
@tfoote just curious: why is `jade` an _old_ distro if `indigo` is a distro?

Reply:
Jade was EOL in May 2017 Indigo is an LTS on Trusty so it's valid through April 2019

http://wiki.ros.org/Distributions